src/Homework5/max_div_exp_of_two.py

```diff
@@ -0,0 +1,13 @@
+"""Вводится число найти его максимальный делитель,
+являющийся степенью двойки. 10(2) 16(16), 12(4).
+"""
+
+
+def div_exp_bit_length(num):
+    div = num.bit_length() - 1
+    while num % (1 << div):
```

Comment:
it can be checked with & it seems

Reply:
I`m not so familiar with these operators, unfortunately, but i will grow!